src/App.tsx

```diff
@@ -15,48 +17,105 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortField {
+  None,
+  Length,
+  Alphabet,
+}
```

Comment:
It's better to set value for enum, becauls by default it set number. Now your enum fully looks like this one
```
enum SortField {
  None = 0,
  Length = 1,
  Alphabet = 2,
}
```
```suggestion
enum SortField {
  None = 'none',
  Length = 'length',
  Alphabet = 'alphabet',
}
```